.github/CODEOWNERS

```diff
@@ -13,3 +13,5 @@
 # Reviewers for layout-related code
 /components/layout_2020 @mrobinson
 /components/layout @mrobinson
+/components/layout_2020 @Loirooriol
+/components/layout @Loirooriol
```

Comment:
I think you have to add your username next to mine for this to work properly: https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#codeowners-syntax

Reply:
Done